### DISCUSSION TITLE
Cant add custom certificate

### DISCUSSION THREAD

**Discussion by OpenK54** [ZoeyVid/NPMplus#257] [Q&A] [answered]
When I try add a custom certificate I always get the error " Key files protected with a passphrase are not supported." But I do know for sure that the key is not protected. The certificate is valid I used it with serveral services and also with the original npm without issues. Any ideas?

**Comment by Zoey2936** [comment]
if you do `docker logs nginx-proxy-manager` do you have something like `Result Validation Error: Validation timed out. This could be due to the key being passphrase-protected.` in your logs?

**Reply by OpenK54** [reply]
I solved the problem. 

the extention box was ticked, but no password was actually set.
![image1](https://user-images.githubusercontent.com/132947170/236949095-45c9fec9-e15d-4106-b5f6-9c15d04eb1fe.jpg)

**Reply by OpenK54** [reply]
I unticket the box and everything worked.

**Reply by Zoey2936** [reply]
but do you have something like this in your logs?

**Reply by OpenK54** [reply]
Yes, I tried to creat a certificate which didn't work just to test it. But I do have my own. And yes, there is no log entry that shows the error in installing the custom certificate. Anyways I worked one time and I could copy the data folder. For me erverything ist o.k. But let me know I can send you logfiles if you want to do some resarche on the issue. Thanks for your support! Cheers!

**Reply by Zoey2936** [reply]
![grafik](https://github.com/ZoeyVid/nginx-proxy-manager/assets/75573284/c190e258-6722-49f1-b320-77a4b744bac8)
you mean this blue line?

**Reply by OpenK54** [reply]
Yes, exactly.

**Reply by Zoey2936** [reply]
this line is always shown, it is not error, it is more like hint for the end users, maybe it looks a bit like an error... The line is the same on the original npm

**Reply by OpenK54** [reply]
o.k. So, I have no idea what the problem was then. Just glad it worked once:-)

**Reply by Zoey2936** [reply]
could you please retry and give it a minute or so?

**Reply by OpenK54** [reply]
will do later and report back.

**Reply by Zoey2936** [reply]
thanks

**Reply by Zoey2936** [reply]
> ![grafik](https://user-images.githubusercontent.com/75573284/237150685-c190e258-6722-49f1-b320-77a4b744bac8.png) you mean this blue line?

I changed the desing a bit:
![grafik](https://github.com/ZoeyVid/nginx-proxy-manager/assets/75573284/a035eb77-5547-4ca8-843d-080befcc0442)

**Comment by OpenK54** [comment]
With a new instance it doesn't work again or anymore:-)

**Comment by OpenK54** [comment]
So, I finally don't know what the issue is. It worked, no it doesn't anymore.

**Reply by Zoey2936** [reply]
are those certs self signed?

**Reply by OpenK54** [reply]
No, it is an official certificate.

![image](https://user-images.githubusercontent.com/132947170/237059829-7437cb0a-214a-4ddd-be52-0962aa5bba50.png)

**Reply by OpenK54** [reply]
But I could solve the problem with copying the data folder from the proxy server with the custom certificate.

**Comment by OpenK54** [comment]
I tested and get this error:

![grafik](https://github.com/ZoeyVid/nginx-proxy-manager/assets/132947170/fa73443d-33b3-4791-8dd8-27419c05caa3)

**Reply by Zoey2936** [reply]
do you have some errors in your browser console? or can you tra with adifferent browser

**Reply by OpenK54** [reply]
I made the certificate new and it worked. So I don't now what the problem was. Thanks for your effort!

**Reply by Zoey2936** [reply]
Interessting... I still habe no Idea why IT works sonetimes and sometimes not, but if it works now, it should be good

**Comment by OpenK54** [comment, marked as answer]
Still happy that it worked once and I just can copy the data folder.
